### PR TITLE
font-freefont-ttf: Inclusion of homepage and metainfo.xml

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Adding `homepage` key to `package.yml`( Part of #411)
- Including `metainfo.xml` (Part of #449)

**Test Plan**

- Build the package
- Run `appstream-builder --packages-dir=. --include-failed -v`
- Check that the package doesn't show up in the `example-failed.xml.gz`
- Verified the added homepage loads information about the package

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/f/font-freefont-ttf/files/freefont.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Solus Developers <copyright@getsol.us> -->
+<component type="font">
+  <id>freefont</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>FreeFont</name>
+  <homepage>https://www.gnu.org/software/freefont/</homepage>
```
Comment (joebonrichie) [inline packages/f/font-freefont-ttf/files/freefont.metainfo.xml:7]: Should be:
```suggestion
  <url type="homepage">https://www.gnu.org/software/freefont/</url>
```

I know you copied it off one of my faulty commits, sorry about that! Please amend the commit and force push.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url

**Review by joebonrichie — CHANGES_REQUESTED** [review]
One small thing

File: packages/f/font-freefont-ttf/files/freefont.metainfo.xml (hunk 1 above)
Comment (malfisya) [reply packages/f/font-freefont-ttf/files/freefont.metainfo.xml]: Done

**Review by ReillyBrogan — APPROVED** [review]
LGTM, Thanks!

**Pull request merged**